src/components/MarkdownProvider/components/Typography.tsx

```diff
@@ -12,14 +12,17 @@ import { XXL, XL, LG } from '@zendeskgarden/react-typography';
 
 const headerSpacing = (p: ThemeProps<DefaultTheme>) => {
   return css`
-    margin-top: ${p.theme.space.md};
+    margin-top: ${p.theme.space.lg};
     margin-bottom: ${p.theme.space.sm};
-    padding-bottom: ${p.theme.space.xs};
+    font-weight: ${p.theme.fontWeights.semibold};
   `;
 };
 
 export const StyledH2 = styled(XXL).attrs({ tag: 'h2' })`
   ${headerSpacing}
+  margin-top: ${p => p.theme.space.xxl};
+  border-bottom: solid 1px ${getColor('grey', 300, p => p.theme)};
```

Comment:
Use `p.theme.borders.sm` for `solid 1px`

Reply:
Looks like you'll need something like the following to correct the TS build errors:
```
border-bottom: ${p => `${p.theme.borders.sm} ${getColor('grey', 300, p.theme)}`};
```